RecoParticleFlow/PFClusterProducer/plugins/SimMappers/RealisticSimClusterMapper.h

```diff
@@ -47,6 +50,9 @@ class RealisticSimClusterMapper : public InitialClusteringStepBase {
   const float exclusiveFraction_ = 0.7f;
   const bool maxDistanceFilter_ = false;
   const float maxDistance_ = 10.f;
+  const float maxDforTimingSquared_ = 4.0f;
```

Comment:
The number of parameters is growing, and their names also get a bit more complicate.
I think that you can avoid future mistakes and problems if you switch since now to the fillDescription() method for them